### ISSUE TITLE
Support Protractor 3.x.x

### ISSUE THREAD

**Issue by tomca32** [NickTomlin/protractor-flake#17]
Would like to use this with the latest version of Protractor. When installing protractor-flake this is the error message:

```
npm ERR! peerinvalid The package protractor@3.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer protractor-flake@1.0.2 wants protractor@2.x
```

**Comment by NickTomlin** [comment]
Ah, great catch! I totally forgot to update this. I have updated the dep to 2.x and greater. This has been published as`1.0.3` please let me know if that has not resolved this for you, thanks!

**Comment by tomca32** [comment]
Great! Thanks for the quick update.
